INSTALL.md

```diff
@@ -88,11 +90,33 @@ git clone https://github.com/oneapi-src/oneTBB.git
 cd oneTBB
 # Create binary directory for out-of-source build
 mkdir build && cd build
-# Configure. Customize CMAKE_INSTALL_PREFIX and disable TBB_TEST to avoid tests build
+# Configure: customize CMAKE_INSTALL_PREFIX and disable TBB_TEST to avoid tests build
 cmake -DCMAKE_INSTALL_PREFIX=/tmp/my_installed_onetbb -DTBB_TEST=OFF ..
 # Build
 cmake --build .
 # Install
 cmake --install .
 # Well done! Your installed oneTBB is in /tmp/my_installed_onetbb
 ```
+
+### Multi-configuration generators
+
+The following example demonstrates how to install oneTBB for multi-configuration generators such as Visual Studio*. 
+
+Choose the configuration during the build and install steps:
+```batch
+REM Do our experiments in C:\temp
+cd C:\temp
```

Comment:
If I'm not mistaken this directory does not exists by default

Reply:
Fixed